### PR TITLE
Updating Navigation bar

### PR DESCRIPTION
**Pull request by jposi**
Updated Marketplace link, removed Agency Lead from CSP view

### CODE REVIEW

**Comment by mgwalker** [comment]
LGTM!  🚀

**Comment by lauraGgit** [comment]
Unfortunately, this isn't consistent with `fedramp.gov` and it would be better if the internal nav links use internal the router controller instead of the full url.

**Comment by lauraGgit** [comment]
@jposi updated the nav again.

File: src/templates/components/navigation.html [hunk 1]
```diff
@@ -27,14 +27,14 @@
...
-                        <li><a href="https://www.fedramp.gov/participate/3paos/a2la-accreditation/" title="A2LA Accreditation">A2LA Accreditation</a></li>
+                        <li><a href="https://www.fedramp.gov/participate/assessors/a2la-accreditation/" title="A2LA Accreditation">A2LA Accreditation</a></li>
                     </ul>
                 </li>
                 <li><a href="https://www.fedramp.gov/provide-public-comment/" title="Provide Public Comment">Provide Public Comment</a></li>
             </ul>
         </li>
         <li class="dropdown">
-            <a href="#"
+            <a href="javascript:;"
```
Comment (mgwalker) [inline src/templates/components/navigation.html:37]: Ah-ha, this makes sense.  Curious about `javascript:;` - can it be an empty `href` instead?  Not a blocker/breaker in either case, though.

Reference:
https://docs.angularjs.org/api/ng/directive/a

**Pull request merged**